explorer/rest/rest/__init__.py

```diff
@@ -25,6 +25,7 @@ def setup_nem_facade(app):
 	app.config.from_envvar('EXPLORER_REST_SETTINGS')
 	config = configparser.ConfigParser()
 	db_path = Path(app.config.get('DATABASE_CONFIG_FILEPATH'))
+	network = str(Path(app.config.get('NETWORK'))).upper()
```

Comment:
why are you wrapping this in Path ?

Reply:
instead of querying the network type, I just added `NETWORK` in my config files, to determine `mainnet` or `testnet`.

So, when initiating the project, it can immediately identify the network type.

Reply:
yes, i understand, but i am asking about `Path`. why can you not just do `app.config.get('NETWORK').upper()` ?

Reply:
ahh, I misunderstood, but you are right `Path` is not needed 🙏🏼